pkgs/development/libraries/srt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake, openssl
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "srt-${version}";
```

Comment:
I'm new to `nix` and I'm wondering, when is a `pname` attribute necessary? I notice it in some packages, but not in others. Shouldn't it be here?

Reply:
It is now possible to specify `version` and `pname`  instead of specifying `name`.

Reply:
OK, switched to that.